app/views/additional_details/submissions/_submission_info.html.haml

```diff
@@ -9,6 +9,10 @@
             %p.list-group-item-text
               - if item.item_type == 'multiple_dropdown'
                 = qr.content.delete('[]').delete('\"')
+              - elsif item.item_type == 'time'
+                = convert_time(qr.content)
+              - elsif item.item_type == 'checkbox'
+                = qr.content.gsub(/["\[\]]/, '')
```

Comment:
I would label that a refactor and make another pull request.  Not necessary to proceed.  In the same thought you could change 'qr.content.gsub' to another method call similar to convert_time.